spec/models/connection_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Connection do
+  it { should belong_to(:guide).class_name 'User' }
+  it { should belong_to(:follower).class_name 'User' }
+end
+
+describe '#guide_name' do
+  it 'returns the name of the guide in a social connection' do
+    guide = Fabricate :user
+    connection = Fabricate :connection, guide: guide
+    expect(connection.guide_name).to eq guide.full_name
+  end
+
+  describe '#guide_queue_size' do
```

Comment:
This certainly does not belong under `describe '#guide_name'`. Check your indentation and blocks.